connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConfig.java

```diff
@@ -250,9 +251,17 @@
             + "user requests to reset the set of active topics per connector.";
     protected static final boolean TOPIC_TRACKING_ALLOW_RESET_DEFAULT = true;
 
+    public static final String TOPIC_CREATION_ENABLE_CONFIG = "topic.creation.enable";
+    protected static final String TOPIC_CREATION_ENABLE_DOC = "If set to true, it allows "
+            + "source connectors to create topics by specifying topic creation properties "
+            + "with the prefix `" + TOPIC_CREATION_PREFIX + "`. Each task will use an "
+            + "admin client to create its topics and will not depend on the Kafka brokers "
+            + "to create topics automatically.";
```

Comment:
The "source connectors to create topics by specifying..." seems strange, since source connectors don't actually create topics and even if they did they wouldn't do so by specifying anything. Maybe:
```suggestion
    protected static final String TOPIC_CREATION_ENABLE_DOC = "Whether to allow "
            + "automatic creation of topics used by source connectors, when source connector "
            + "are configured with `" + TOPIC_CREATION_PREFIX + "` properties. Each task will use an "
            + "admin client to create its topics and will not depend on the Kafka brokers "
            + "to create topics automatically.";
```

Reply:
Accepted the suggestion here and I'll return to fix the typo in `source connector`